UVa/BasicRemains.java

```diff
@@ -0,0 +1,41 @@
+
+/*
+Given a base b and two non-negative base b integers
```

Comment:
You actually make a very good point. I'll leave it up to @kdn251 to have the final say on if he wants to have a consistent style or "googleability".